resource-watch/resource-watch
# shorten long URL with bitly in share modal option

## Overview
Shorten/hash long URLs when sharing.

## Testing instructions
Go to Data / Explore DataSets
Click on share button of map controls panel.
The URL that appears as "Public url to share" should be shorten by bitly

## Pivotal task
https://www.pivotaltracker.com/n/projects/1374154/stories/161620064.

---

## Checklist before submitting
[ ] Title: Don't forget to make it clear for everyone, even for people that don't know about the feature/bug.
[ ] Meaningful commits and code rebased on `develop`.
